common/src/main/java/com/redhat/cloud/notifications/models/event/TestEventHelper.java

```diff
@@ -0,0 +1,123 @@
+package com.redhat.cloud.notifications.models.event;
+
+import com.redhat.cloud.notifications.ingress.Action;
+import com.redhat.cloud.notifications.ingress.Context;
+import com.redhat.cloud.notifications.ingress.Event;
+import com.redhat.cloud.notifications.ingress.Metadata;
+import com.redhat.cloud.notifications.ingress.Payload;
+import com.redhat.cloud.notifications.ingress.Recipient;
+
+import java.time.Clock;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+/**
+ * The purpose of this class is to help creating test events for testing the
+ * integrations that the clients have set up. The used constants are also
+ * exposed for any required checks or tests that might need them. For more
+ * information, take a look at <a href="https://issues.redhat.com/browse/RHCLOUD-22470">RHCLOUD-22470</a>.
+ */
+public class TestEventHelper {
+    /**
+     * Regular action test data.
+     */
+    public static final String TEST_ACTION_CONTEXT_TEST_EVENT = "test-action-context-test-event";
+    public static final boolean TEST_ACTION_CONTEXT_TEST_EVENT_VALUE = true;
+    public static final String TEST_ACTION_CONTEXT_ENDPOINT_ID = "test-action-context-endpoint-id";
+    public static final String TEST_ACTION_METADATA_KEY = "test-metadata-key";
+    public static final String TEST_ACTION_METADATA_VALUE = "test-metadata-value";
+    public static final String TEST_ACTION_PAYLOAD_KEY = "test-payload-key";
+    public static final String TEST_ACTION_PAYLOAD_VALUE = "test-payload-value";
+    public static final String TEST_ACTION_RECIPIENT = "test-recipient-1";
+    public static final String TEST_ACTION_VERSION = "0.0.0";
```

Comment:
I'm not sure of the correct version we should use here, but `0.0.0` doesn't feel right. To be confirmed.

Reply:
Actually, we don't even need to specify the version in this class. You can simply remove it.